com.unity.shadergraph/Editor/Data/Nodes/SlotValue.cs

```diff
@@ -63,14 +63,76 @@ enum ConcreteSlotValueTypePopupName
         Vector2 = ConcreteSlotValueType.Vector2,
         Float = ConcreteSlotValueType.Vector1, // This is currently the only renaming we need - rename Vector1 to Float
         Boolean = ConcreteSlotValueType.Boolean,
-        VirtualTexture = ConcreteSlotValueType.VirtualTexture
+        VirtualTexture = ConcreteSlotValueType.VirtualTexture,
+
+        BareSamplerState = 1000 + ConcreteSlotValueType.SamplerState,
```

Comment:
is this just a priority thing to make them display last? If so, could you just throw in a comment?

Reply:
Oh just saw later how thats used. Still want a comment explaining though

Reply:
Yeah, want these at the bottom of the list and the rest of the enum values to match ConcreteSlotValueType.

Hmm...  I don't think the 1000 + is actually necessary...  this isn't a serialized type, so we don't need to preserve old values in any way, it's only used to populate the popup menu selecting the type in a custom function node.

Doesn't hurt though, and calls out that they are kind of special.

Reply:
I can add a comment noting that these are special.